Annotations/AddCustomStampInCustomizedButton/MainWindow.xaml.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Syncfusion.Pdf.Graphics;
+using Syncfusion.Windows.PdfViewer;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace AddCustomStampInCustomizedButton
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private bool checkAddAnnotation = false;
+        public MainWindow()
+        {
+            InitializeComponent();
+            pdfViewer.Load("../../Data/F#.pdf");
+            pdfViewer.PageClicked += PdfViewer_PageClicked;
+        }
+
+        private void PdfViewer_PageClicked(object sender, Syncfusion.Windows.PdfViewer.PageClickedEventArgs args)
+        {
+            if(checkAddAnnotation)
```

Comment:
Can you provided proper naming for this Boolean ? as CanAddAnnotation

Reply:
Done